docs/reference/api-overview.md

```diff
@@ -8,55 +8,64 @@ approvers:
 ---
 
 {% capture overview %}
-This page contains an overview of the Kubernetes API. 
+This page provides an overview of the Kubernetes API. 
 {% endcapture %}
 
 {% capture body %}
-The REST API is the fundamental fabric of Kubernetes. All operations and communications between components are REST API calls handled by the API Server, including external user commands. Consequently, everything in the Kubernetes
+The REST API is the fundamental fabric of Kubernetes. All operations and communications between components, and external user commands are REST API calls that the API Server handles. Consequently, everything in the Kubernetes
 platform is treated as an API object and has a corresponding entry in the
 [API](/docs/api-reference/{{page.version}}/).
 
 Most operations can be performed through the
-[kubectl](/docs/user-guide/kubectl-overview/) command-line interface or other
-command-line tools, such as [kubeadm](/docs/admin/kubeadm/), which in turn use
-the API. However, the API can also be accessed directly using REST calls.
+[kubectl](/docs/reference/kubectl/overview/) command-line interface or other
+command-line tools, such as [kubeadm](/docs/reference/setup-tools/kubeadm/kubeadm/), which in turn use
+the API. However, you can also access the API directly using REST calls.
 
 Consider using one of the [client libraries](/docs/reference/client-libraries/)
 if you are writing an application using the Kubernetes API.
 
 ## API versioning
 
-To make it easier to eliminate fields or restructure resource representations, Kubernetes supports
-multiple API versions, each at a different API path, such as `/api/v1` or
+To eliminate fields or restructure resource representations, Kubernetes supports
+multiple API versions, each at a different API path. For example: `/api/v1` or
 `/apis/extensions/v1beta1`.
 
-The version is set at the API level rather than at the resource or field level to ensure that the API presents a clear, consistent view of system resources and behavior, and to enable controlling access to end-of-life and/or experimental APIs. The JSON and Protobuf serialization schemas follow the same guidelines for schema changes; all descriptions below cover both formats.
+The version is set at the API level rather than at the resource or field level to:
+- Ensure that the API presents a clear and consistent view of system resources and behavior.
+- Enable control access to end-of-life and/or experimental APIs.
 
-Note that API versioning and software versioning are only indirectly related.  The [API and release
-versioning proposal](https://git.k8s.io/community/contributors/design-proposals/release/versioning.md) describes the relationship between API versioning and software versioning.
+The JSON and Protobuf serialization schemas follow the same guidelines for schema changes.
+
+The following descriptions cover both formats:
```

Comment:
I'm not sure this sentence makes sense with the next element being a call-out note. I'm not 100% sure on the intent, but I think it's implying that the alpha/API structure stuff below applies to gRPC as well as JSON/Protobuf serialization schemas. I think this might be better just outright reworded to that end, and drop the `:` trying to lead into a following section